doc/phpmatrix.md

```diff
@@ -0,0 +1,36 @@
+# Dolibarr PHP support
```

Comment:
We must avoid to add document files that need to be maintained into source code. This duplicate the information and it is often not updated. A link in the readme.md seems better